.golangci.yml

```diff
@@ -116,57 +87,97 @@ linters:
     - whitespace
     # - wrapcheck # Fix and enable
     - wsl
-issues:
-  exclude-rules:
-    # Separating explicit var declarations by blank lines seems excessive.
-    - linters:
-        - wsl
-      text: "declarations should never be cuddled"
+  settings:
+    gocritic:
+      disabled-checks:
+        - ifElseChain
+        - unnamedResult
+      enabled-tags:
+        - diagnostic
+        - opinionated
+        - performance
+        - style
+    gocyclo:
+      min-complexity: 15
+    govet:
+      enable:
+        - fieldalignment
+    lll:
+      line-length: 140
+    wsl:
+      # Separating explicit var declarations by blank lines seems excessive.
+      allow-cuddle-declarations: true
+  exclusions:
+    generated: lax
+    presets:
+      - comments
+      - common-false-positives
+      - legacy
+      - std-error-handling
+    rules:
+      # Separating explicit var declarations by blank lines seems excessive.
+      - linters:
+          - wsl
+        text: declarations should never be cuddled
 
-    # This rule incorrectly flags code where two logically-related lines are
-    # followed by an if conditional to check their err return.
-    - linters:
-        - wsl
-      text: "only one cuddle assignment allowed before if statement"
+      # This rule incorrectly flags code where two logically-related lines are
+      # followed by an if conditional to check their err return.
+      - linters:
+          - wsl
+        text: only one cuddle assignment allowed before if statement
 
-    # This rule incorrectly flags code where two logically-related lines are
-    # used to create vars that are consumed in a loop.
-    - linters:
-        - wsl
-      text: "only one cuddle assignment allowed before range statement"
+      # This rule incorrectly flags code where two logically-related lines are
+      # used to create vars that are consumed in a loop.
+      - linters:
+          - wsl
+        text: only one cuddle assignment allowed before range statement
 
-    # Allow dot-imports for Gomega BDD directives per idiomatic Gomega
-    - linters:
-        - revive
-        - stylecheck
-      text: "dot imports"
-      source: "gomega"
+      - linters:
+          - revive
+          - staticcheck
+        text: dot imports
+        source: gomega
 
-    # Allow dot-imports for Ginkgo BDD directives per idiomatic Ginkgo
-    - linters:
-        - revive
-        - stylecheck
-      text: "dot imports"
-      source: "ginkgo"
+      # Allow dot-imports for Ginkgo BDD directives per idiomatic Ginkgo
```

Comment:
This should be done by configuring `revive` and `staticcheck`, see https://github.com/submariner-io/shipyard/pull/1963/files#diff-6179837f7df53a6f05c522b6b7bb566d484d5465d9894fb04910dd08bb40dcc9R39